src/utils.js

```diff
@@ -466,7 +466,7 @@ exports.insertPixel = function (url) {
     } catch (e) {
     }
   };
-  this.insertElement(img);
+  exports.insertElement(img);
```

Comment:
Updated the other `insertElement` reference as well. I'd like to go through utils and update remaining `this` references at some point, or convert to ES module syntax entirely, but for now just scoped this PR to `insertElement` references